### PR TITLE
Adds repo and version commands

### PR DESCRIPTION
**Pull request by mtfd**
Not sure if anyone cares about these two commands but it was some low hanging fruit that was bugging me.

### CODE REVIEW

File: dongerdong.py [hunk 1]
```diff
@@ -271,6 +271,14 @@ def on_message(self, target, source, message):
                         c += 1
                         if c == 4:
                             break
+                elif command == "version" and not self.gameRunning:
+                    try:
+                        ver = subprocess.check_output(["git", "describe"]).strip()
```
Comment (Polsaker) [inline dongerdong.py]: You could .decode() this to make the output prettier

File: config.example [hunk 2]
```diff
@@ -8,5 +8,6 @@
     "nickserv_password": "yourpasswd",
     "auxchans": [],
     "extendedcommands": ["jaden","excuse","dong"],
-    "topmodifier": 0.05
+    "topmodifier": 0.05,
+    "repolink": "https://github.com/tonyravioli/dongerdong"
```
Comment (Polsaker) [inline config.example]: Why would this be a config variable? If somebody forks this project, they can just edit the sauce.

File: dongerdong.py (hunk 1 above)
Comment (mtfd) [inline dongerdong.py]: Donezo in commit 46a38e7

File: config.example (hunk 2 above)
Comment (mtfd) [inline config.example]: Yeah but why have the repo link in the source? I think config is exactly what this type of thing is for.

Comment (Polsaker) [inline config.example]: The config is for stuff the end user has to edit to make the bot run

Comment (mtfd) [inline config.example]: I can update to not use a config var if that's the preference but like I said, in my projects this is exactly something I would consider a config element. No biggie tho

**Pull request closed without merging**